zipline/pipeline/term.py

```diff
@@ -24,6 +24,8 @@
 from zipline.utils.sentinel import sentinel
 
 
+ALLOWED_WINDOWED_INPUTS = ['Returns']
```

Comment:
This feels too brittle to me.  What do you think about just having `window_safe` attribute on `Term`, which defaults to False, and is set to True on `BoundColumn`s and on any Factors that we've explicitly marked as safe (other good candidates for window_safe expressions would be the outputs of `Factor.rank` and `Factor.zscore`).